### DISCUSSION TITLE
Estatuto em cartório

### DISCUSSION THREAD

**Discussion by anadulce** [apyb/comunidade#163] [Pergunta]
Uma instituição nos solicitou uma cópia do estatuto e suas alterações em que consta o registro em cartório para notas de empenho.
Teria como vocês me fornecerem esta documentação. Muito obrigada!

@apyb/diretoria

**Comment by cecivieira** [comment]
@anicelysantos

**Comment by anicelysantos** [comment]
Aqui @anadulce https://drive.google.com/drive/u/0/folders/1XOS9S--wQdl_wUeltnpbdrwZVCSDMABu

**Reply by anadulce** [reply]
Muuuito obrigada, @anicelysantos !!
